cinema_service/urls.py

```diff
@@ -1,9 +1,12 @@
+from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
 
+from cinema_service import settings
+
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Serving media files using `static()` is suitable for development but not for production. In a production environment, consider using a dedicated media server or cloud storage service to serve media files efficiently and securely.